Ladybird/Headless/Test.cpp

```diff
@@ -453,7 +453,7 @@ ErrorOr<void> run_tests(Core::AnonymousBuffer const& theme, Gfx::IntSize window_
                 outln("");
 
             Core::deferred_invoke([&]() mutable {
-                if (s_skipped_tests.contains_slow(test.input_path))
+                if (s_skipped_tests.contains_slow(MUST(FileSystem::real_path(test.input_path))))
```

Comment:
Is this path not already the real path? We already do `FileSystem::real_path` in `collect_dump_tests` and `collect_ref_tests`